examples/src/examples/FiabInteractiveEngineExample.java

```diff
@@ -47,6 +43,9 @@ public static void main(String[] args) throws InterruptedException, JsonProcessi
       calcParameters.setFisettingsdocument(FIAB_SETTINGS_DOCUMENT);
       calcParameters.setMsl(FIAB_MSL);
       ApiResponse<Void> response = apiInstance.runCalculationWithHttpInfo(calcParameters);
+      //Comment the above line and uncomment the below lines to add cache control configuration. Results are by default cached for 12 hours; Setting max-stale=300 will fetch a cached result which is 5 minutes older.
+      //String cache_control="max-stale=300";
+      //ApiResponse<Void> response = apiInstance.runCalculationWithHttpInfo(cache_control, calcParameters);
```

Comment:
It think that we don't support cache_control for FIAB. Can you please crosscheck?

Reply:
Yes, I have removed the lines and updated the code.